app/views/aid_applications/verifications/edit.html.erb

```diff
@@ -44,19 +42,29 @@
             <%= t('.application_number.edit_contact') %>
           <% end %>
           <%= f.cfa_checkbox :contact_method_confirmed, t('.application_number.contact_method') %>
-        </div>
 
-        <hr>
+          <hr>
 
-        <div class="form-card__content">
           <h2><%= t('.next_steps.title') %></h2>
-          <ul class="list--numbered">
+          <ol class="list--numbered">
             <li><%= t('.next_steps.step1') %></li>
             <li><%= t('.next_steps.step2') %></li>
             <li><%= t('.next_steps.step3') %></li>
             <li><%= t('.next_steps.step4') %></li>
             <li><em><%= t('.next_steps.step5') %></em></li>
-          </ul>
+          </ol>
+
+            <%= f.cfa_radio_set(
+                    :card_receipt_method,
+                    label_text: t('.receive_card.title'),
+                    help_text: t('.receive_card.subtitle'),
+                    collection: [
+                        {value: 'Pick up', label: t('.receive_card.options.pick_up')},
+                        {value: 'Deliver in person', label: t('.receive_card.options.deliver')},
+                        {value: 'Mail', label: t('.receive_card.options.mail')},
+                        {value: 'Decide later', label: t('.receive_card.options.decide_later')}
```

Comment:
Please make the values be more computery. e.g. `decide_later`. You could turn them into constants inside of the AidApplication to be extra strict.

Reply:
Beyond that, this looks awesome 👍